python/langsmith/evaluation/_runner.py

```diff
@@ -1323,14 +1323,15 @@ def get_summary_scores(self) -> Dict[str, List[dict]]:
     # Private methods
 
     def _predict(
-        self, target: TARGET_T, /, max_concurrency: Optional[int] = None
+        self, target: TARGET_T, /, max_concurrency: Optional[int] = None,
     ) -> Generator[_ForwardResults, None, None]:
         """Run the target function on the examples."""
-        fn = _ensure_traceable(target)
+        fn, with_attachments = _ensure_traceable(target)
```

Comment:
can we make the attachments check its own function? doesnt' seem very related to ensure_traceable

Reply:
also out of curiosity if someone does
```python
@traceable
def target(inputs, attachments): ...
 ```
are we properly tracing attachments? or are they going to be traced as part of the regular run inputs?

Reply:
I think they are being traced correctly? I ran some experiments and the attachments show up in the run, not sure if that is what you mean